### PR TITLE
Created new test collection for Ldap integration

### PR DESCRIPTION
**Pull request by ehardon**
- created new postman environment for the test collection
- added json file with data for the tests

### CODE REVIEW

File: test/postman/Ldap-synchronization.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+								"protocol",
+								"openid-connect",
+								"token"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Generate Keycloak ldapUser access token",
```
Comment (kconkas) [inline test/postman/Ldap-synchronization.postman_collection.json]: At this point, if I'm not mistaken, we've got a running KeyCloak instance pointing to LDAP server but haven't yet run the LDAP user synchronisation. Will the user be able to get a token?
(also, this is probably something that more suits AUTH-162)

File: test/postman/Ldap-synchronization.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+									"});",
+									"",
+									"",
+									"tests['Ldap user has required username'] = jsonData.username === data.value",
+									"tests['Ldap user has required firstName'] = jsonData.firstName === data.value",
+									"tests['Ldap user has required lastName'] = jsonData.lastName === data.value",
+									"tests['Ldap user has required email'] = jsonData.email === data.value",
+									"tests['Ldap user has required attributes'] = jsonData.attributes === data.value",
+									"tests['Ldap user has required access'] = jsonData.access === data.value",
+									""
```
Comment (kconkas) [inline test/postman/Ldap-synchronization.postman_collection.json]: 👍 this proves attributes synched

File: test/postman/Ldap-synchronization.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,394 @@
...
+								"openid-connect",
+								"token"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Check Ldap users Sync",
+					"event": [
```
Comment (kconkas) [inline test/postman/Ldap-synchronization.postman_collection.json]: Again, this test happens before the user synch has been executed. While KeyCloak may indeed import LDAP users at startup (I couldn't find a reference to this in the documentation) perhaps this test would be more reliable if executed _after_ the synch? Just to prove this I suggest running this collection against a KeyCloak instance in which these LDAP users don't exist.

**Comment by kconkas** [comment]
This is really a minor note and doesn't affect functionality but if all files were to be named consistently (e.g. all in lower case) that could prevent potential issues on case-insensitive operating systems (e.g. `Ldap-synchronization.postman_collection.json` versus `ldap-synchronization.postman_collection.json`)

File: test/postman/Ldap-synchronization.postman_collection.json (hunk 1 above)
Comment (ehardon) [reply test/postman/Ldap-synchronization.postman_collection.json]: yes, you're right! I'll better remove this one

File: test/postman/Ldap-synchronization.postman_collection.json (hunk 3 above)
Comment (ehardon) [reply test/postman/Ldap-synchronization.postman_collection.json]: I'll try doing that!

**Pull request merged**